app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,67 @@
+<section class="recipe-div">
+  <h1>Recipe <%= @recipe.name %></h1>
+  <h3>Preparation Time: <%= @recipe.preparation_time %></h3>
+  <h3>Cooking Time: 
+    <%= @recipe.cooking_time %>
+  </h3>
+  <% if @recipe.owner?(current_user) %>
+  <div class="toggle">
+    <h4>
+      <%if @recipe.public == true %>
+      Public
+      <% else %>
+      Private
+      <% end %>
+    </h4>
+    <label class="switch">
+      <% if @recipe.public %>
+      <input type="checkbox" checked />
+      <% else %>
+      <input type="checkbox" />
+      <% end %>
+      <span class="slider round"></span>
+    </label>
+  </div>
```

Comment:
- [ ] It will be great to ensure that when the user who created the recipe makes the recipe private, it does not display on the public list recipe. This functionality is not working as I can still see the recipes that I have switched them to private still being listed in the public recipe lists

Reply:
@peterrobert  if this functionality has to be implemented, then i have to update the recipe. but in the requirements, it states I have to create CRUD except Update. 

A recipes list (with all CRUD implementation, except for 'update').

this project requirement says I have to display the toggle button. it doesn't say anything about updating the recipe details.

Recipe details:

    Should display a toggle button that allows for a recipe to be made public or private.

Reply:
Kindly as you can see in the screenshot below, the toggle button should allow us to make it either public or private. Kindly check here: ⬇️ 
![Screenshot 2023-05-26 at 16 11 00](https://github.com/ram1117/recipe-app/assets/32361963/44a056ba-a92c-4416-ae68-ffbb7fe2cf06)

Reply:
The toggle button should allow us to make the recipe either public or private.

Reply:
so it should: 

1. be displayed
2.  Allow the recipe to be made public or private